build/build/src/enso.rs

```diff
@@ -198,7 +203,6 @@ impl BuiltEnso {
             _ => None,
         };
 
-        let std_tests = crate::paths::discover_standard_library_tests(&paths.repo_root)?;
```

Comment:
If this line disappeared, how does the next line still compile??

Reply:
Having weird merge issues with this line. I will put it back.

Reply:
Oh you moved it to line 139 in a previous MR